src/components/MultiHopTrade/components/TradeInput/components/TradeInputHeader.tsx

```diff
@@ -0,0 +1,122 @@
+  const activeQuote = useAppSelector(selectActiveQuote)
+  const activeSwapperName = useAppSelector(selectActiveSwapperName)
+  const isTradeQuoteApiQueryPending = useAppSelector(selectIsTradeQuoteApiQueryPending)
+
+  const pollingInterval = useMemo(() => {
+    if (!activeSwapperName) return DEFAULT_GET_TRADE_QUOTE_POLLING_INTERVAL
+    return swappers[activeSwapperName]?.pollingInterval ?? DEFAULT_GET_TRADE_QUOTE_POLLING_INTERVAL
+  }, [activeSwapperName])
+
+  const isRefetching = useMemo(
+    () => Boolean(activeSwapperName && isTradeQuoteApiQueryPending[activeSwapperName] === true),
+    [activeSwapperName, isTradeQuoteApiQueryPending],
+  )
+
+  return (
+    <>
+      {activeQuote && (isCompact || isSmallerThanXl) && (
+        <CountdownSpinner isLoading={isLoading || isRefetching} initialTimeMs={pollingInterval} />
+      )}
+      <SlippagePopover />
+    </>
+  )
+}
+
+export const TradeInputHeader = ({
+  initialTab,
+  isCompact,
+  isLoading,
+  onChangeTab,
+}: FakeTabHeaderProps) => {
+  const translate = useTranslate()
+  const [selectedTab, setSelectedTab] = useState<TradeInputTab>(initialTab)
+
+  const enableBridgeClaims = useFeatureFlag('ArbitrumBridgeClaims')
+
+  const handleClickTrade = useCallback(() => {
+    setSelectedTab(TradeInputTab.Trade)
+    onChangeTab(TradeInputTab.Trade)
+  }, [onChangeTab])
+
+  const handleClickClaim = useCallback(() => {
+    setSelectedTab(TradeInputTab.Claim)
+    onChangeTab(TradeInputTab.Claim)
+  }, [onChangeTab])
+
+  const rightComponent = useMemo(() => {
+    return (() => {
+      switch (selectedTab) {
+        case TradeInputTab.Trade:
+          return <TradeInputHeaderRightComponent isLoading={isLoading} isCompact={isCompact} />
+        case TradeInputTab.Claim:
+          return null
+        default:
+          assertUnreachable(selectedTab)
+      }
+    })()
+  }, [selectedTab, isLoading, isCompact])
+
+  return (
+    <CardHeader px={6}>
+      <Flex alignItems='center' justifyContent='space-between'>
+        <Flex gap={4}>
+          <Heading
+            as='h5'
+            fontSize='md'
+            color={selectedTab !== TradeInputTab.Trade ? 'text.subtle' : undefined}
+            onClick={handleClickTrade}
+            cursor={selectedTab !== TradeInputTab.Trade ? 'pointer' : undefined}
+          >
+            {translate('navBar.trade')}
+          </Heading>
+          {enableBridgeClaims && (
+            <Heading
+              as='h5'
+              fontSize='md'
+              color={selectedTab !== TradeInputTab.Claim ? 'text.subtle' : undefined}
+              onClick={handleClickClaim}
+              cursor={selectedTab !== TradeInputTab.Claim ? 'pointer' : undefined}
+            >
+              {translate('bridge.claims')}
+            </Heading>
+          )}
+        </Flex>
+        <Flex gap={2} alignItems='center' height={6}>
+          {rightComponent}
+        </Flex>
+      </Flex>
+    </CardHeader>
+  )
+}
```

Comment:
thought: definitely something we can abstract even one level upper to avoid UI/UX discrepancies because a similar tab workflow is used in:

- Loans
- Swapper
- rFOX
- LP

Reply:
Good point, though this was originally named `FakeTabHeader` because the other instances of this style of tab header actually use the built-in chakra tabs whereas this is attempting to replicate the visual elements without the tab integration

Reply:
In my previous experiences, we were implementing the tabs systems using contexts so we could retrieve any active tab from any level of depth, wdyt about this? 

Anyway, this is just an open discussion, not blocking